docs/features/parameters.md

```diff
@@ -43,7 +43,41 @@ public class ReportJob : IJob
 }
 ```
 
+It's also possible to configure a job with only a parameter and no cron expression.
```

Comment:
I am really not sure if this is too implicit. 
So, that we configure a job and trigger an instant job. How could the user override it? What if the user wants an explicit `null`? How can we distinguish this behavior?

Reply:
Configuring a Job with a parameter and allowing its override at runtime is already supported.

See https://github.com/NCronJob-Dev/NCronJob/blob/ed0f4ce2eea702832906ae94df6ca245b23fc762/tests/NCronJob.Tests/IntegrationTests.cs#L109-L125

However, handling `null` explicitly as a parameter value looks like a new requirement.

I have no idea how to weight how useful that could be (I've haven't designed any job in such a way).
If you believe this should be handled, let's create an issue and try and make it happen.

Reply:
Test https://github.com/NCronJob-Dev/NCronJob/blob/e93ccfead09ed521cdf67086bf249a8962d8861d/tests/NCronJob.Tests/RuntimeJobRegistryTests.cs#L312 has been added in #249

Tracked through #251